recipes-core/readline/readline/inputrc

```diff
@@ -28,23 +28,23 @@ $if mode=emacs
 "\e[4~": end-of-line
 
 # allow the use of the Delete/Insert keys
-# "\e[3~": delete-char
+"\e[3~": delete-char
 # "\e[2~": quoted-insert
 
-# mappings for "page up" and "page down" to step to the beginning/end 
+# mappings for "page up" and "page down" to step to the beginning/end
 # of the history
 # "\e[5~": beginning-of-history
 # "\e[6~": end-of-history
 
 # alternate mappings for "page up" and "page down" to search the history
-# "\e[5~": history-search-backward
-# "\e[6~": history-search-forward
+"\e[5~": history-search-backward
+"\e[6~": history-search-forward
 
 # mappings for Ctrl-left-arrow and Ctrl-right-arrow for word moving
-# "\e[5C": forward-word
-# "\e[5D": backward-word
-# "\e\e[C": forward-word
-# "\e\e[D": backward-word
+"\e[5C": forward-word
+"\e[5D": backward-word
+"\e\e[C": forward-word
+"\e\e[D": backward-word
```

Comment:
I use alt-f and alt-b for forward-word and backward and don't use ctrl-arrow.  I turned it on since it was enabled on my Fedora box... Not sure if it matters for alt-f or not.

I'll respin with